src/Mod/Sketcher/Gui/CommandConstraints.cpp

```diff
@@ -1252,6 +1254,9 @@ class DrawSketchHandlerDimension : public DrawSketchHandler
         if (specialConstraint == SpecialConstraint::LineOr2PointsDistance)
             updateDistanceType(onSketchPos);
 
+        if (specialConstraint == SpecialConstraint::LinesAngle)
+            updateAngle(onSketchPos);
+
```

Comment:
Looks like there's something wrong with the pre-commit CI (or its application of clang-format) -- I thought we set it up so that curly braces were required even on one-liners.